NEW_MODULE_TECH_EVAL.MD

```diff
@@ -0,0 +1,73 @@
+
+# New Module Technical Evaluation Processes
+
+## Overview
+This document outlines the processes for **technical** evaluation of module for inclusion in a FOLIO release.  The following activities are covered:
+1. Submission
+1. Evaluation
+1. Review
+1. Feedback
+1. Approval/Rejection
+
+### Supporting documentation
+* Details of the JIRA workflow are captured in a separate document:  (insert link). 
+* Technical evaluation is part of a larger process involving the other FOLIO Councils (PC/CC).  A description of the full process can be found here:  (insert link). 
```

Comment:
We use the full names for these councils below. Could we use them here as well (or define these abbreviations during their first use)?